src/main/scala/ai/privado/languageEngine/ruby/processor/RubyProcessor.scala

```diff
@@ -62,6 +64,12 @@ object RubyProcessor {
         logger.info("Enhancing Ruby graph")
         logger.debug("Running custom passes")
 
+        if (config.skipDownloadDependencies) {
```

Comment:
shouldn't this be `!config.skipDownloadDependencies`?

Reply:
yes right, while testing i forgot to add `!` at last.